src/communicate/main.py

```diff
@@ -802,6 +967,24 @@ def get_user_role(cur, user_id):
 
     return user_role
 
+#issue#425
+def get_user_sys_roles(cur, user_id):
```

Comment:
is this function in use? seems not

Reply:
Will use in the next edition when I will set filter for testers instead of my username.